fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -1310,15 +1312,15 @@ public void testCreateNewBinaryWithChecksumSHAandMD5() throws MalformedRdfExcept
     @Test(expected = CannotCreateResourceException.class)
     public void testLDPRNotImplemented() throws Exception {
         final var resource = setResource(Container.class);
-        when(resourceFactory.getResource(mockTransaction, pathId.resolve("x"))).thenReturn(resource);
+        when(resourceFactory.getResource(mockTransaction.getId(), pathId.resolve("x"))).thenReturn(resource);
```

Comment:
Seems like this and the other `getResource(mockTransaction.getId()` calls can be reverted to just `getResource(mockTransaction` if that is more accurate to the real usage